Win11ThemeTest/DatepickerTest.cs

```diff
@@ -0,0 +1,524 @@
+            if (parts[1] == hParts[0])
+            {
+                dayBtn = dayButtons[44];
+                dayBtnString = dayButtons[44].Name;
+                parts = dayBtnString.Split(' ');
+            }
+            if (parts[1] != hParts[0])
+            {
+                dayBtn.Click();
+                calBtn.Click();
+                Wait.UntilInputIsProcessed();
+            }
+            dayBtnString = dayBtnString.Remove(0, 3);
+            AutomationElement headerBtnNew = calWidget.FindFirstChild(cf => cf.ByAutomationId("PART_HeaderButton"));
+            Assert.That(headerBtnNew.Name, Is.EqualTo(dayBtnString));
+            calBtn.Click();
+        }
+        #endregion
+
+        //test keyboard input for date- valid date
+        [Test]
+        public void datePickers5_ValidKeyboardInput()
+        {
+            Assert.IsNotNull(datepicker);
+            datepicker.Focus();
+            DateTime sdate = new DateTime(2024, 09, 21);
+            Keyboard.Type("2024/09/21");
+            Keyboard.Press(FlaUI.Core.WindowsAPI.VirtualKeyShort.ENTER);
+            Keyboard.Release(FlaUI.Core.WindowsAPI.VirtualKeyShort.ENTER);
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(1000));
+            Assert.IsNotNull(dtTextBox);
+            Assert.That(dtTextBox.Text, Is.EqualTo(sdate.ToString("dd MMMM yyyy")));
+        }
+
+        //test keyboard input for date- invalid date
+        [Test]
+        public void datePickers6_InvalidKeyboardInput()
+        {
+            Assert.IsNotNull(datepicker);
+            string beforeDate = datepicker.Patterns.Value.Pattern.Value;
+            datepicker.Focus();
+            Wait.UntilInputIsProcessed();
+            Keyboard.Type("2024/20/21");
+            Keyboard.Press(FlaUI.Core.WindowsAPI.VirtualKeyShort.ENTER);
+            Keyboard.Release(FlaUI.Core.WindowsAPI.VirtualKeyShort.ENTER);
+            Wait.UntilInputIsProcessed();
+            string afterDate = datepicker.Patterns.Value.Pattern.Value;
+            Assert.That(beforeDate, Is.EqualTo(afterDate));
+        }
+
+        //right click on datepicker textbox-menu items copy, cut and paste
+        [Test]
+        public void datePickers7_RightClick()
+        {
+            Assert.IsNotNull(dtTextBox);
+            Assert.IsNotNull(dtPickerWindow);
+            dtTextBox.Focus();
+            dtTextBox.RightClick();
+            dtMenuCopy = dtPickerWindow.FindFirstDescendant(cf => cf.ByName("Copy")).AsMenuItem();
+            dtMenuPaste = dtPickerWindow.FindFirstDescendant(cf => cf.ByName("Paste")).AsMenuItem();
+            dtMenuCut = dtPickerWindow.FindFirstDescendant(cf => cf.ByName("Cut")).AsMenuItem();
+            Assert.IsNotNull(dtMenuCopy);
+            Assert.IsNotNull(dtMenuPaste);
+            Assert.IsNotNull(dtMenuCut);
+            Mouse.Click();
+        }
+
+        //test for copy and paste menu action items
+        [Test]
+        public void datePickers8_datepickerCopyPaste()
```

Comment:
check this test, it is failing giving exception -


  Message: 
  Expected string length 12 but was 13. Strings differ at index 0.
  Expected: "comboBoxTest"
  But was:  "02 April 2024"